src/sentry/api/endpoints/event_owners.py

```diff
@@ -21,7 +22,11 @@ def get(self, request, project, event_id):
         :auth: required
         """
 
-        event = Event.objects.from_event_id(event_id, project.id)
+        use_snuba = options.get('snuba.events-queries.enabled')
+
+        event_cls = SnubaEvent if use_snuba else Event
+
+        event = event_cls.objects.from_event_id(event_id, project.id)
```

Comment:
Would it be outlandish to make `Event.objects.from_event_id` do this logic internally? That might help you play fewer rounds of whac-a-mole.